codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecompressorTest.java

```diff
@@ -53,10 +53,11 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
         response.headers().set(HttpHeaderNames.CONTENT_TYPE, "application/json;charset=UTF-8");
         response.headers().set(HttpHeaderNames.TRANSFER_ENCODING, HttpHeaderValues.CHUNKED);
 
-        Assert.assertTrue(channel.writeInbound(response));
+        Assert.assertFalse(channel.writeInbound(response));
 
+        channel.read();
         // we triggered read explicitly
-        Assert.assertEquals(1, readCalled.get());
+        Assert.assertEquals(2, readCalled.get());
```

Comment:
@carryxyh can you explain why these changes are needed ?

Reply:
channel.read -> tail -> readCalled.incrementAndGet() -> head.read -> 
![image](https://user-images.githubusercontent.com/12047600/79867230-9ad1b800-8410-11ea-8297-7673f5d5edf0.png) -> readCalled.incrementAndGet().

means `readCalled.incrementAndGet` will called twice in this chain...

(I find it difficult to understand the whole process described in words. Perhaps the call process is more direct..)